README.md

```diff
@@ -1,3 +1,124 @@
 # cloud-sql-go-connector
 *Warning*: This project is experimental, and is not an officially supported 
 Google product.
+
+The _Cloud SQL Go Connector_ provides IAM authorization to an app's connections 
+to a Cloud SQL instance. It provides connection level authorization only; it 
+_does not_ provide a network path to an instance that doesn't already exist 
+(ie. you will still be unable to connect via Private IP without access to the 
```

Comment:
i.e.

Reply:
connect via Private IP, or connect to a Private IP database? The word "via" seems strange to me here.

Reply:
The database can have both a private and a public IP. I switched to "to an instance's Private IP" for clarity